### PR TITLE
perbaikan admin

### PR DESCRIPTION
**Pull request by faiz9877**
cek col

### CODE REVIEW

**Comment by aldiskatel** [comment]
follow ini
https://github.com/faiz9877/web.github.io/commit/52e4e2f570fd50478dacab579b9d66c3c242d865

**Comment by faiz9877** [comment]
ok

**Comment by aldiskatel** [comment]
aku kah menggawi atau km?
https://github.com/faiz9877/web.github.io/pull/23#issuecomment-486314233

**Comment by faiz9877** [comment]
wait I'm still not giving up

File: admin/tampil_admin.php [hunk 1]
```diff
@@ -20,10 +20,10 @@
     $halaman = 5;
     $page = isset($_GET["halaman"]) ? (int)$_GET["halaman"] : 1;
     $mulai = ($page>1) ? ($page * $halaman) - $halaman : 0;
-    $result = mysqli_query($link, "SELECT*FROM users");
+    $result = mysqli_query($koneksi,"select from username,created_at from users");
     $total = mysqli_num_rows($result);
     $pages = ceil($total/$halaman);s
```
Comment (aldiskatel) [inline admin/tampil_admin.php]: ada huruf facking S

**Comment by faiz9877** [comment]
merge Di

**Comment by faiz9877** [comment]
pertama kali d terima

**Pull request merged**